onadata/libs/utils/common_tags.py

```diff
@@ -159,3 +159,26 @@
 XLS_DATE_FIELDS = ['date', 'today']
 SUBMISSION_REVIEW = 'submission_review'
 XLS_DATETIME_FIELDS = ['start', 'end', 'dateTime', '_submission_time']
+
+# Cache names used in the open_data viewet
```

Comment:
is this only related to the `open_data` viewset or are these all the general metadata fields? also typo in viewset

Reply:
It is comprised of the general metadata fields, yes.

I believe it is only the tableau functionality that removes metadata tags
So there is no where else that these tags are combined together in a list except here
I may however be wrong on this.

Reply:
OK, in that case I think you can remove this comment, the variable name is explanatory

Reply:
Done